### ISSUE TITLE
Dependencies: Avoid projects to upgrade jasmine-core to 2.6.0

### ISSUE THREAD

**Issue by jonaswjs** [LaxarJS/generator-laxarjs2#11]
Because of some errors when using `jasmine-core 2.6.0 ` in PhantomJS, we should use `jasmine-core 2.5.2` as dependency.

Replace the caret `"jasmine-core": "^2.5.2",` with the tilde `"jasmine-core": "~2.5.2",` in the `dependencies.json` for the `app`.

One example with the shop-demo:

The tests of the article-browser-widget resolves in

```
PhantomJS 2.1.1 (Linux 0.0.0) The article-browser-widget subscribes to didReplace events of the articles resource FAILED

TypeError: undefined is not a function (evaluating 'allKeys[i].match(/^[0-9]+$/)') in node_modules/jasmine-core/lib/jasmine-core/jasmine.js (line 2576)

webpack:///application/widgets/article-browser-widget/spec/article-browser-widget.spec.js:41:32 <- application/widgets/article-browser-widget/spec/article-browser-widget.spec.js:17820:27
```

https://github.com/LaxarJS/shop-demo/blob/master-2.x/application/widgets/article-browser-widget/spec/article-browser-widget.spec.js#L41

**Comment by x1B** [comment]
@jonaswjs it seems like the Jasmine team fixed (some of) the issues with [v2.6.1](https://github.com/jasmine/jasmine/blob/master/release_notes/2.6.1.md) -- Do you think this is still needed?

**Comment by alex3683** [comment]
`2.6.4` is now the current version of Jasmine. I think I'll set the version to `^2.6.4`. Okay? @x1B @jonaswjs

**Comment by x1B** [comment]
@alex3683 agreed

**Comment by alex3683** [comment]
Fixed on `master` (`v2.0.0`).
